portfolio/src/main/webapp/script.js

```diff
@@ -25,6 +25,12 @@ const COMMENT_CONTAINER = 'comment-container';
 const SELECTED_INDEX = 'selectedIndex';
 const COMMENT_AMOUNT = 'amount';
 const DATA_SERVLET = '/data';
+const SORT = 'sort';
+const AMOUNT_SELECTED_INDEX = 'amountSelectedIndex';
+const SORT_SELECTED_INDEX = 'sortSelectedIndex';
+const P_TAG = 'p';
+const DIV_TAG = 'div';
+const END_OF_TIMESTAMP = 21;
```

Comment:
It is not clear why 21? Can you add a comment explaining?

Reply:
Done!

Reply:
Is there a method that exists to get the timestamp without milliseconds? It would be simpler than using this.

Reply:
I think I would have to individually get the day, month, year, hour, and seconds. I doubled checked the string and realized that the timezone name is also included so I updated the comment to reflect that. 

Would it be better to individually get the different values or is it okay to leave it as is?

Reply:
It would be cleaner to do it the other way, but it isn't a big deal. You can just add a TODO.